packages/web-components/fast-foundation/docs/api-report.md

```diff
@@ -200,6 +203,29 @@ export interface DecoratorDesignSystemPropertyConfiguration extends Omit<Decorat
     default: any;
 }
 
+// @public
+export class DelegatesFocus {
+    ariaAtomic: "true" | "false";
```

Comment:
Need to re-build docs

Reply:
aha

Reply:
This should be good and everything should be up to date